Src/Sarif.PartnerMatcher.Function/Rules/rules.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
remove this file, since its a test rule.

Reply:
deleted